content/en/docs/reference/setup-tools/kubeadm/implementation-details.md

```diff
@@ -674,7 +674,11 @@ upgrades the control plane node where it's run. The steps it performs are:
   for the kubelet to restart the components if the files have changed.
 - Uploads the updated kubeadm and kubelet configurations to the cluster in the `kubeadm-config`
   and the `kubelet-config` ConfigMaps (both in the `kube-system` namespace).
-- Writes updated kubelet configuration for this node in `/var/lib/kubelet/config.yaml`.
+- Writes updated kubelet configuration for this node in `/var/lib/kubelet/config.yaml`,
+  and read the node's `/var/lib/kubelet/instance-config.yaml` file
+  and patch the `containerRuntimeEndpoint` value 
+  from this instance configuration into `/var/lib/kubelet/config.yaml`. 
+  This instance configuration contains the node-specific CRI socket.
```

Comment:
```suggestion
```

making it more generic as we may use the instance-config for other things in the future